### ISSUE TITLE
Incorrect type for arrays in constructor parameters

### ISSUE THREAD

**Issue by frostiq** [dbrainio/Soltsice#10]
Hi!

First of all, thanks for your great library that makes solidity contracts testings mush easier! :) :+1: 

During usage, I found one bug, regarding generated type for arrays in contract constructor parameters.

For example, if we have such contract:
```
pragma solidity ^0.4.18;

contract Test {
    uint[10] array;
  
    function Test(uint[10] _array) {
        array = _array;
    }
}
```

If I generate TypeScript for this contract, I get next `New` method:
```
// tslint:disable-next-line:max-line-length
    static async New(deploymentParams: W3.TC.TxParams, ctorParams?: {_array: BigNumber | number}, w3?: W3, link?: SoltsiceContract[]): Promise<Test> {
        let contract = new Test(deploymentParams, ctorParams, w3, link);
        await contract._instancePromise;
        return contract;
    }
```

So seems like constructor parameters have incorrect type:
`ctorParams?: {_array: BigNumber | number}`

Instead of:
`ctorParams?: {_array: BigNumber[] | number[]}`

Thanks in advance for your attention!

**Comment by buybackoff** [comment]
Will fix `abiTypeToTypeName` in `soltsice.ts` soon. In your case it handles the type via `.stratsWith(uint)` code path, which is wrong. Haven't encountered numeric arrays in our contacts yet, so there is long standing TODO in that function to handle all types. 

BTW, a pull request is very welcome ;)

**Comment by frostiq** [comment]
Please, take a look :)

https://github.com/dbrainio/Soltsice/pull/11

**Comment by buybackoff** [comment]
Fixed in #11
